src/main/java/pl/trainingapp/controller/ExerciseController.java

```diff
@@ -1,13 +1,48 @@
 package pl.trainingapp.controller;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 import pl.trainingapp.services.ExerciseService;
 
+
 @RestController
 @RequiredArgsConstructor
 public class ExerciseController {
 
     private final ExerciseService exerciseService;
 
+    @GetMapping("/exercises")
```

Comment:
You can extract `/exercises` to class level annotation, and use only `/{id}` etc on methods

Reply:
ok, fixed